app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+
+  def index
+  end
+
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
+    @comment.update(user_id: current_user.id)
+    redirect_to post_path(@post)
+  end
+
+  def show
+    @comment = Comment.find(params[:id])
+  end
+
+  def edit
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @comment = Comment.find(params[:id])
+    @comment.update(comment_params)
```

Comment:
Could put a conditional here that current user is the comment's author else have a flash message only authors can edit comments